pay-api/src/pay_api/resources/v1/eft_short_names.py

```diff
@@ -35,11 +41,22 @@ def get_eft_shortnames():
     """Get all eft short name records."""
     current_app.logger.info('<get_eft_shortnames')
 
-    include_all = bool(request.args.get('includeAll', 'false').lower() == 'true')
+    state = request.args.get('state', None)
     page: int = int(request.args.get('page', '1'))
     limit: int = int(request.args.get('limit', '10'))
+    transaction_date = request.args.get('transactionDate', None)
+    deposit_amount = request.args.get('depositAmount', None)
+    deposit_date = request.args.get('depositDate', None)
+    short_name = request.args.get('shortName', None)
 
-    response, status = EFTShortnameService.search(include_all, page, limit), HTTPStatus.OK
+    response, status = EFTShortnameService.search(EFTShortnamesSearch(
+        deposit_date=string_to_date(deposit_date) if deposit_date else None,
+        deposit_amount=Decimal(deposit_amount) * Decimal(100) if deposit_amount else None,
+        short_name=short_name,
+        transaction_date=string_to_date(transaction_date) if transaction_date else None,
```

Comment:
This was an existing utility function I re-used. I will take a look at some of the existing usages and see if it impacts it.

Reply:
it would throw a runtime error if None was put in there fairly certain, I don't think you'd be impacting anything 

![image](https://github.com/bcgov/sbc-pay/assets/3484109/f8001e88-6666-4e92-abae-4c1ebc78dc86)

Reply:
Updated utils function and any usages.